hooks/shapediver/viewer/interaction/useHoverManager.ts

```diff
@@ -0,0 +1,56 @@
+import { IInteractionParameterSettings, MaterialStandardData } from "@shapediver/viewer";
+import { HoverManager } from "@shapediver/viewer.features.interaction";
+import { useRef, useEffect } from "react";
+import { useShapeDiverStoreViewer } from "shared/store/useShapeDiverStoreViewer";
+import { useInteractionEngine } from "./useInteractionEngine";
+
+// #region Functions (1)
+
+/**
+ * Hook allowing to create the hoverManager.
+ * 
+ * @param viewportId 
+ */
+export function useHoverManager(viewportId: string, settings?: IInteractionParameterSettings): {
+	/**
+	 * The hover manager.
+	 */
+    hoverManager?: HoverManager
+} {
+	// get the viewport API
+	const viewportApi = useShapeDiverStoreViewer(state => { return state.viewports[viewportId]; });
```

Comment:
@MajorMeerkatThe3rd I think we don't need the viewportApi here, it should be sufficient to check for the interactionEngine